src/utils/encryption.ts

```diff
@@ -0,0 +1,33 @@
+import crypto from 'crypto'
```

Comment:
`./helpers` lived within `./redux` and was renamed to `./initializers`, which contains initialization functions for Web3 + IPFS.

`./utils` lives outside of redux and provides utilities for IPFS and our data encryption strategy, but does not aggregate its exports to prevent it from becoming a dumping ground. It is simply a folder that contains resource specific utilities. We can rename this to `./helpers` if you want, but same purpose.

Reply:
i vote /helpers cuz i tend to use it in other of our codebases

and i fukin hate `utils`

Reply:
Updated